### PR TITLE
Preparation work for Js perf dashboard

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

File: src/components/DashboardHeader/index.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  },
+  subtitle: {
+      color: '#d1d2d3',
+      fontWeight: '400',
+      margin: '.5rem .6rem .5rem',
+      padding: '.5em 0',
+  },
+};
+
+const DashboardHeader = ({ classes, title, subtitle }) => (
```
Comment (klahnakoski) [inline src/components/DashboardHeader/index.jsx]: Why does DashboardHeader exist?  Is there a plan to use it in other dashboards?  If this component has only one call point, I suggest it stay inlined.  At the very least, give it a name that indicates it is private (use underscore as prefix, like `_DashboardHeader`)

File: src/components/SectionHeader/index.jsx [hunk 2]
```diff
@@ -17,10 +22,12 @@ const styles = () => ({
 });
 
 const SectionHeader = ({ classes, title, subtitle }) => (
```
Comment (klahnakoski) [inline src/components/SectionHeader/index.jsx]: Is the SectionHeader used in places other than <Section>?  This should be inlined, or made private.

File: src/views/JsTeam/index.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    static propTypes = {
+        classes: PropTypes.shape({}).isRequired,
+    };
+
+    render() {
+        const { classes } = this.props;
+        return (
+          <div>
+            <DashboardHeader title='JavaScript Performance dashboard' />
+            <div className={classes.content}>
```
Comment (klahnakoski) [inline src/views/JsTeam/index.jsx]: I assume `classes.content` will not change; why is it a `prop`?

File: src/views/JsTeam/index.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import DashboardHeader from '../../components/DashboardHeader';
+
+const styles = ({
+    content: {
+        margin: '1rem',
+    },
+});
+
+class JsTeam extends Component {
+    static propTypes = {
```
Comment (klahnakoski) [inline src/views/JsTeam/index.jsx]: are we going with 4-space indent, or 2-space indent?  I like the former better.

**Comment by armenzg** [comment]
@klahnakoski I know this will have impact on your other PRs but I want to land it so @aimenbatool will have less CSS madness to deal with when integrating the JS perf dashboard.

If I don't hear back from you by tomorrow (you should really stay away from IRC 😄 ) I will be merging it as is.

**Comment by klahnakoski** [comment]
@armenzg yes, go and merge. I saw this coming.

**Pull request merged**